template/src/index.js

```diff
@@ -1,4 +1,5 @@
 import './main.css'
+import './Stylesheets.elm'
```

Comment:
This kind of stuff should be documented in [User Guide.](https://github.com/halfzebra/create-elm-app/blob/master/template/README.md)